src/http/sr-modules-keys/sr-modules-keys.controller.ts

```diff
@@ -14,13 +14,14 @@ import {
 import { ApiNotFoundResponse, ApiOperation, ApiParam, ApiResponse, ApiTags } from '@nestjs/swagger';
 import { SRModuleKeyListResponse, GroupedByModuleKeyListResponse } from './entities';
 import { SRModulesKeysService } from './sr-modules-keys.service';
-import { ModuleId, KeyQuery, Key } from '../common/entities/';
+import { KeyQuery, Key } from '../common/entities/';
 import { KeysFindBody } from '../common/entities/pubkeys';
 import { TooEarlyResponse } from '../common/entities/http-exceptions';
 import { IsolationLevel } from '@mikro-orm/core';
 import { EntityManager } from '@mikro-orm/knex';
 import * as JSONStream from 'jsonstream';
 import type { FastifyReply } from 'fastify';
+import { ModuleIdPipe } from 'http/common/pipeline/module-id-pipe';
```

Comment:
we use a mix of absolute and relative imports, perhaps we should make up our minds?